pkg/features/kube_features.go

```diff
@@ -751,7 +751,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	LocalStorageCapacityIsolationFSQuotaMonitoring: {Default: false, PreRelease: featuregate.Alpha},
 	NonPreemptingPriority:                          {Default: true, PreRelease: featuregate.Beta},
 	PodOverhead:                                    {Default: true, PreRelease: featuregate.Beta},
-	IPv6DualStack:                                  {Default: false, PreRelease: featuregate.Alpha},
+	IPv6DualStack:                                  {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Also, need to update the comment here to beta: https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L473-L476

Reply:
last open issue, I think?